consensus/istanbul/backend/api.go

```diff
@@ -150,9 +150,21 @@ func (api *API) GetValEnodeTable() (map[string]*vet.ValEnodeEntryInfo, error) {
 
 // GetCurrentRoundState retrieves the current IBFT RoundState
 func (api *API) GetCurrentRoundState() (*core.RoundStateSummary, error) {
+	if api.istanbul.coreStarted {
+		return nil, istanbul.ErrStoppedEngine
+	}
```

Comment:
Is this correct? Seems to be like there should be a `not` here, as in if the core is *not* started, return `ErrStoppedEngine`.
Maybe I'm missing something, but if so maybe a comment would be helpful.

Reply:
heh... you're right!